core/modules/node/node.module

```diff
@@ -1488,6 +1490,12 @@ function node_ranking() {
       // The promote flag is either 0 or 1, which is automatically normalized.
       'score' => 'n.promote',
     ),
+    'title_match' => array(
+      'title' => t('Content title match'),
+      // If the keywords appear in the title, return 1, else 0.
+      'score' => "IF((CONCAT(',', title, ',') REGEXP ',(:words),' > 0), '1','0')",
```

Comment:
The code is Greek to me, but shouldn't this be n.title?

Reply:
Maybe! I'll try that.